device-types/Cisco/WS-C6513.yaml

```diff
@@ -8,14 +8,38 @@ is_full_depth: false
 console-ports:
```

Comment:
Is this a parent chassis?? If so, then this needs to be reworked so that it has `subdevice_role: parent`

Reply:
6500 is a modular switch